packages/datepickers/src/elements/Datepicker/Datepicker.tsx

```diff
@@ -30,66 +30,49 @@ import { DatepickerContext } from './utils/useDatepickerContext';
 import { StyledMenu, StyledMenuWrapper } from '../../styled';
 
 export interface IDatepickerProps {
-  /**
-   * The selected date to display
-   */
+  /** Sets the selected date */
   value?: Date;
   /**
-   * Returns the parsed date
+   * Handles changes to the selected date
+   *
+   * @param {Date} date The selected date to use
    */
   onChange?: (date: Date) => void;
   /**
-   * Allows customization of date formatting within input.
+   * Formats the date to a string. The result is displayed in the date input box when a date is selected.
+   *
+   * @param {Date} date The date to format
+   * @returns {string}
    */
   formatDate?: (date: Date) => string;
-  /**
-   * Accepts [all valid Intl locales](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation)
+  /** Sets the locale. This accepts
+   * [all valid `Intl` locales](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation).
    */
   locale?: string;
-  /**
-   * Minimum value. Users are still able to manually enter a date below this value.
-   */
+  /** Sets the minimum value, but users can still enter a date below this value */
   minValue?: Date;
-  /**
-   * Maximum value. Users are still able to manually enter a date above this value.
-   */
+  /** Sets the maximum value, but users can still enter a date above this value */
   maxValue?: Date;
-  /**
-   * Show compact styling
-   */
+  /** Applies compact styling */
   isCompact?: boolean;
   /**
-   * Override default date parsing. Receives a localized input value and returns a `Date` object.
+   * Parses the date with a custom format and returns a `Date` object
```

Comment:
Simplify to:

> Parses the date with a custom format

and move "...and returns a `Date` object" to the returns description as:

> ```@returns {Date} a valid `Date` object```

Reply:
Fixed in latest push.